src/rust-crypto/verification.ts

```diff
@@ -653,11 +667,22 @@ export class RustSASVerifier extends BaseRustVerifer<RustSdkCryptoJs.Sas> implem
         if (req) {
             await this.outgoingRequestProcessor.makeOutgoingRequest(req);
         }
+        this.started = true;
```

Comment:
set this *before* `await this.outgoingRequestProcessor.makeOutgoingRequest(req);`, in case the new `Verification` arrives while we're sending the `m.key.verification.start`? Not sure if that's possible, but moving the assignment would save me worrying about it.

Reply:
I originally had it before, but moved it to after, in response to https://github.com/matrix-org/matrix-js-sdk/pull/4004#discussion_r1462921558  I've now moved it back to before, but wrapped the acceptance in a try-catch to address the original comment.

However, thinking about this more, I don't think we even need the ~`.started`~ `.accepted` variable.  We will only ever call `.setInner` if we started a verification, but lost the tie breaker.  So in that case, we know we definitely do want to send an acceptance.  Does that sound right?

Reply:
yeah, that sounds right.

(It would be nice if we could introspect the old `RustSdkCryptoJs.Sas` to double-check that it thinks we have `accept`ed, but I that doesn't seem to be exposed in the API, so let's just assume it.)